Pcap++/src/PcapRemoteDeviceList.cpp

```diff
@@ -121,7 +119,7 @@ PcapRemoteDevice* PcapRemoteDeviceList::getRemoteDeviceByIP(IPv4Address ip4Addr)
 				continue;
 			}
 
-			if (currAddr->s_addr == ip4Addr.toInAddr()->s_addr)
+			if (memcmp(currAddr, ip4Addr.toBytes(), sizeof(currAddr)) == 0)

```

Comment:
why using `memcmp` here instead of comparing `currAddr->s_addr` with `ip4Addr.toUInt()`?

Reply:
The same as for `PcapLiveDeviceList.cpp`

Reply:
`memcmp` was removed